tests/errors/cpp_no_constructor.pyx

```diff
@@ -9,5 +9,5 @@ cdef extern from *:
 new Foo(1, 2)
 
 _ERRORS = u"""
-9:7: Call with wrong number of arguments (expected 1, got 2)
+9:7: Call with wrong number of arguments (expected 0, got 2)
```

Comment:
Yeah, this is what I meant with deterministic output above. This change seems arbitrary.